libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Security/SFUserAccountManager.h

```diff
@@ -99,64 +128,18 @@ FOUNDATION_EXTERN NSString * const kSFLoginHostChangedNotificationUpdatedHostKey
  */
 @property (nonatomic, strong, nullable) SFUserAccount *currentUser;
 
-/** The user identity for the temporary user account.
- */
-@property (nonatomic, readonly, nullable) SFUserAccountIdentity *temporaryUserIdentity;
-
-/** The "temporary" account user.  Useful for determining whether there's a valid user context.
- */
-@property (nonatomic, readonly, nullable) SFUserAccount *temporaryUser;
-
-/** Returns YES if the application supports anonymous user, no otherwise.
- 
- Note: the application must add the kSFUserAccountSupportAnonymousUsage value
- to its Info.plist file in order to enable this flag.
- */
-@property (nonatomic, readonly) BOOL supportsAnonymousUser;
-
-/** Returns YES if the application wants the anonymous user to be
-  created automatically at startup, no otherwise.
-  
-  Note: the application must add the kSFUserAccountSupportAnonymousUsage value
-  to its Info.plist file in order to enable this flag.
-  */
-@property (nonatomic, readonly) BOOL autocreateAnonymousUser;
-
-/** Returns the anonymous user or nil if none exists
-  */
-@property (nonatomic, strong, readonly, nullable) SFUserAccount *anonymousUser;
-
 /** Returns YES if the current user is anonymous, no otherwise
   */
 @property (nonatomic, readonly, getter=isCurrentUserAnonymous) BOOL currentUserAnonymous;
 
 /**  Convenience property to retrieve the current user's identity.
  */
-@property (nonatomic, readonly, nullable) SFUserAccountIdentity *currentUserIdentity;
+@property (readonly, nonatomic, nullable) SFUserAccountIdentity *currentUserIdentity;
 
 /**  Convenience property to retrieve the current user's communityId.
  This property is an alias for `currentUser.communityId`
  */
-@property (nonatomic, readonly, nullable) NSString *currentCommunityId;
-
-/** An NSArray of all the SFUserAccount instances for the app.
- */
-@property (nonatomic, readonly) NSArray<SFUserAccount*> *allUserAccounts;
-
-/** Returns all the user identities sorted by Org ID and User ID.
- */
-@property (nonatomic, readonly) NSArray<SFUserAccountIdentity*> *allUserIdentities;
-
-/** The most recently active user identity. Note that this may be temporarily
- different from currentUser if the user associated with the activeUserIdentity
- is removed from the accounts list.
- */
-@property (nonatomic, copy, nullable) SFUserAccountIdentity *activeUserIdentity;
-
-/** The most recently active community ID. Set when a user
- is changed and stored to disk for retrieval after bootup
- */
-@property (nonatomic, copy, nullable) NSString *activeCommunityId;
+@property (nonatomic, nullable) NSString *currentCommunityId;
```

Comment:
Also, [Line 565](https://github.com/forcedotcom/SalesforceMobileSDK-iOS/pull/1939/files#diff-ff01086677618e4fc2ffb0721a725b3fR565) of the implementation defines a retrieval of this value from `NSUserDefaults`, with no equivalent setter.